src/core/env/uat/terraform.tfvars

```diff
@@ -592,7 +592,7 @@ eventhubs = [
   },
   {
     name              = "rtd-trx"
-    partitions        = 1
+    partitions        = 16
```

Comment:
This change should  be moved in another PR.. the queue is hosted in another domain

Reply:
Moved change to https://github.com/pagopa/cstar-infrastructure/pull/857